app/controllers/admin/orders_controller.rb

```diff
@@ -1,10 +1,57 @@
 class Admin::OrdersController < Admin::AdminController
   before_action :load_order_info, only: :index
+  before_action :load_order, except: :index
+  before_action :order_params, only: :update
+
   def index
     @pagy, @orders = pagy Order.all.order_newest
   end
 
+  def update
+    ActiveRecord::Base.transaction do
+      result = @order.update(status: order_params["status"])
```

Comment:
```suggestion
      result = @order.update!(status: order_params["status"])
```
Khi em dùng transaction thì cần phải có exception thì mới có thể rollback các bản ghi đc